### PR TITLE
Example model

### PR DESCRIPTION
**Pull request by kafoso**
I added a model containing examples. To be expanded in the future.

Awaits PR: https://github.com/Starbow/website/pull/9
Next PR: https://github.com/Starbow/website/pull/11

**Please don't close or merge this pull request; let the person who created the pull request do this.**

### CODE REVIEW

File: server/mvc/models/DevExamples.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+var config,
+  thinky;
+
+module.exports = function(){
+  var self = this;
+
+  this.whatIsOnePlusTwo = function(){
+    return 1 + 2;
+  };
+  this.promimeMeEverythingWillBeAlright = function(){
```
Comment (theevenprime2) [inline server/mvc/models/DevExamples.js]: "promise"?

**Comment by theevenprime2** [comment]
As I'm probably the least experienced on this team, I'm really appreciating these examples.  Looks great, thanks!

File: server/mvc/models/DevExamples.js (hunk 1 above)
Comment (Sikian) [inline server/mvc/models/DevExamples.js]: It will, Kas, we'll rule the RTS world.
<3

Comment (kafoso) [inline server/mvc/models/DevExamples.js]: I misspelled it ;)

**Pull request merged**